### ISSUE TITLE
Guide to install with CLI is obsolete

### ISSUE THREAD

**Issue by ryckowski** [syndesisio/syndesis.io#135]
There's guide https://syndesis.io/quickstart/ how to install with CLI but the path to syndesis-dev.yml no longer exists and it seems that support for installing from  templates has been removed in favor of the operator

**Comment by KurtStam** [comment]
@ryckowski yes - well we still have an undocumented feature to generate a template from the operator code. Let's

* document this feature
* update the install instructions.

For now you should be able to follow the 'rocket launch': https://github.com/syndesisio/syndesis-quickstarts/blob/master/README.md#2-install-a-local-mini-cloud-called-minishift

**Comment by ryckowski** [comment]
Any tip how generate a template ?
Actually I'd like to install custom build on my openshift server. Have an idea to push images to quay.io and change source in the template 
I did 'rocket launch' many many times time so far but never success to install stable version like 1.8.12 for weeks.
